fluentd/configs.d/openshift/filter-k8s-meta.conf

```diff
@@ -20,3 +20,10 @@
   preserve_json_log "#{ENV['PRESERVE_JSON_LOG'] || 'true'}"
   json_fields "#{ENV['JSON_FIELDS'] || 'log,MESSAGE'}"
 </filter>
+
+<filter kubernetes.var.log.containers.eventrouter-** kubernetes.var.log.containers.cluster-logging-eventrouter-**>
```

Comment:
`cluster-logging-eventrouter` is to workaround a "bug" in the current documentation - https://docs.openshift.com/container-platform/4.1/logging/efk-logging-eventrouter.html - it really should be `eventrouter`